### ISSUE TITLE
Sign Up and Login pages

### ISSUE THREAD

**Issue by Syed0206** [JAYESHBATRA/Virtuo-Learn#206]
I would like to add Sign Up and Login Pages to the website @JAYESH-BATRA please assign me this issue under GSSOC'24

**Comment by JAYESHBATRA** [comment]
@Syed0206 , for the login and signup functionality, we aim for the signup popup to appear exclusively for quizzes, simulations, and 3D visualizations, rather than on the homepage. We can implement this either from scratch or opt for Clerk, which would be a preferable choice.

**Comment by Syed0206** [comment]
Yes Sir I will make pop up to appear exclusively for quizzes,simulations, and 3D visualizations, rather than on the homepage.

**Comment by mtg718** [comment]
Hi, @JAYESH-BATRA  please assign this issue to me under GSSoc'24.

**Comment by Tanishka023** [comment]
Hey!! @JAYESHBATRA I can do this right away, please assign this to me.
